test/unit/storageFacade.test.ts

```diff
@@ -334,3 +335,8 @@ test('Test 13: Sorted Dashboard Data (packages)', async () => {
     console.log('Test 13 output:\n', JSON.stringify(cves, null, 2));
   });
 });
+
+//Test 14: purge
+test('Test 14: Test Purge', async () => {
+  purge_session(sessionId);
+});
```

Comment:
only thing I'd say is if we wanted to, for these tests it would be nice to have a pre-test setup and post-test cleanup function. But nbd since the previous tests will set up everything